lib/scsi.h

```diff
@@ -884,6 +903,144 @@ struct UnmapBlockDescriptor {
 } ABSL_ATTRIBUTE_PACKED;
 static_assert(sizeof(UnmapBlockDescriptor) == 16);
 
+// SCSI Reference Manual Table 372
+// https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+struct CachingModePage {
```

Comment:
Please double check the order of bit-fields in structs in this PR.

Reply:
Great catch, there were a number of fields out of order